src/main/java/frc/robot/subsystems/coraller/Angler.java

```diff
@@ -57,7 +71,7 @@ public double getSetPoint() {
     return setPointAngle;
   }
 
-  public boolean isAtCorallerSetPoint() {
+  public boolean isAtAnglerSetPoint() {
```

Comment:
Since we are in the context of the Angler, I think we can omit the subsystem from the method name. 
```suggestion
  public boolean isAtSetPoint() {
```

Reply:
I don't think we need to simplify it